protocols/discord.py

```diff
@@ -99,6 +103,12 @@ def _burst_guild(self, guild):
         pylink_netobj.uplink = None
         pylink_netobj._guild_name = guild.name
 
+        # First, burst the webhook agent, if enabled
+        self.webhook_agent_uid = None
+        webhook_agent = pylink_netobj.serverdata.get('webhooks_agent', False)
+        if webhook_agent:
+            self.webhook_agent_uid = self._burst_webhook_agent(webhook_agent, guild, pylink_netobj)
```

Comment:
This should be stored in the pylink_netobj object, since the UID may differ between subservers.